extensions/cornerstone/src/commandsModule.ts

```diff
@@ -646,6 +647,26 @@ function commandsModule({
       }
       stateSyncService.store(storeState);
     },
+
+    /**
+     * Toggle Image Overlays
+     *
+     * @param param0
+     */
+    toggleImageOverlay: ({ toggledState }) => {
```

Comment:
I'm surprised we have a separate method for this here. It is a toggle similar to Crosshairs and every other toggle we have. We should treat it the same